ui: Updates to auth-method show page

kaxcode opened the pull request:
- Update Certificate to be monospaced
- Add empty states for claim and list claim mappings
- Update the styling of empty state actions block

[Demo Link](https://consul-ui-staging-ndzws32au-hashicorp.vercel.app/ui/dc1/acls/auth-methods/capacitor-0/auth-method#CONSUL_ACLS_ENABLE=1)

<img width="1101" alt="Screen Shot 2021-03-18 at 9 32 11 AM" src="https://user-images.githubusercontent.com/19161242/111634374-d31e7980-87cc-11eb-954a-5de36bc9caea.png">

<img width="1581" alt="Screen Shot 2021-03-17 at 5 04 40 PM" src="https://user-images.githubusercontent.com/19161242/111539771-a45cc080-8744-11eb-916a-d2f9f6f9944a.png">

johncowen commented on ui/packages/consul-ui/app/styles/components/empty-state/skin.scss:
```diff
@@ -2,9 +2,6 @@
-%empty-state > ul {
-  border-color: $gray-300;
-}
 %empty-state-header {
```
Are we ok removing this line app wide? I had a look in the various figma files and it seems like sometimes this line is there and sometimes its not, all in all I'm not massively clear to me whether this needs removing app wide or only on certain empty states, @jnwright can you let us know whats required here?

jnwright replied:
It can be removed 👍🏼

johncowen reviewed (approved):
🔥 awesome

I left one comment but its mainly for Jasmine to resolve for us.

I suppose the last thing is, if it was me I'd add the carriage returns in the PEM certificate format to the mocked data so its a standard looking cert, that way you get a more realistic idea of what this will look like to users. AFAIK they generally have returns in them.

kaxcode commented:
> I suppose the last thing is, if it was me I'd add the carriage returns in the PEM certificate format to the mocked data so its a standard looking cert, that way you get a more realistic idea of what this will look like to users. AFAIK they generally have returns in them.

@johncowen I've updated the mock data and updated the screen shot in PR. [Demo Link](https://consul-ui-staging-bmp4y72eb-hashicorp.vercel.app/ui/dc1/acls/auth-methods/application-4/auth-method)
